src/app/pages/data-protection/components/data-protection-dashboard/data-protection-dashboard.component.ts

```diff
@@ -809,55 +795,57 @@ export class DataProtectionDashboardComponent implements OnInit, OnDestroy {
       const log = row.job && row.job.logs_excerpt ? row.job.logs_excerpt : null;
       if (error === null && log === null) {
         this.dialog.Info(globalHelptext.noLogDilaog.title, globalHelptext.noLogDilaog.message, '500px', 'info', true);
       }
 
-      const dialog_title = T('Task State');
-      const dialog_content =
-        (error ? `<h5>${T('Error')}</h5> <pre>${error}</pre>` : '') +
-        (log ? `<h5>${T('Logs')}</h5> <pre>${log}</pre>` : '');
-
-      if (log) {
-        this.dialog
-          .confirm(
-            dialog_title,
-            dialog_content,
-            true,
-            T('Download Logs'),
-            false,
-            '',
-            '',
-            '',
-            '',
-            false,
-            T('Cancel'),
-            true,
-          )
-          .subscribe((dialog_res) => {
-            if (dialog_res) {
-              const filename = `${row.job.id}.log`;
-              this.ws.call('core.download', ['filesystem.get', [row.job.logs_path], filename]).subscribe(
-                (res) => {
-                  const url = res[1];
-                  const mimetype = 'text/plain';
-                  this.storage.streamDownloadFile(this.http, url, filename, mimetype).subscribe(
-                    (blob) => this.storage.downloadBlob(blob, filename),
-                    (err) => new EntityUtils().handleWSError(this, err),
-                  );
-                },
-                (err) => new EntityUtils().handleWSError(this, err),
-              );
-            }
-          });
-      }
+      this.job.showLogs(row.job);
+
+      // const dialog_title = T('Task State');
+      // const dialog_content =
+      //   (error ? `<h5>${T('Error')}</h5> <pre>${error}</pre>` : '') +
+      //   (log ? `<h5>${T('Logs')}</h5> <pre>${log}</pre>` : '');
+
+      // if (log) {
+      //   this.dialog
+      //     .confirm(
+      //       dialog_title,
+      //       dialog_content,
+      //       true,
+      //       T('Download Logs'),
+      //       false,
+      //       '',
+      //       '',
+      //       '',
+      //       '',
+      //       false,
+      //       T('Cancel'),
+      //       true,
+      //     )
+      //     .subscribe((dialog_res) => {
+      //       if (dialog_res) {
+      //         const filename = `${row.job.id}.log`;
+      //         this.ws.call('core.download', ['filesystem.get', [row.job.logs_path], filename]).subscribe(
+      //           (res) => {
+      //             const url = res[1];
+      //             const mimetype = 'text/plain';
+      //             this.storage.streamDownloadFile(this.http, url, filename, mimetype).subscribe(
+      //               (blob) => this.storage.downloadBlob(blob, filename),
+      //               (err) => new EntityUtils().handleWSError(this, err),
+      //             );
+      //           },
+      //           (err) => new EntityUtils().handleWSError(this, err),
```

Comment:
What happened here?

Reply:
I accidentally added uncompleted changes — marked PR as a draft until getting it fixed.